.travis.yml

```diff
@@ -25,7 +24,7 @@ before_script:
   - Xvfb :99 -screen 0 1024x768x24 -extension RANDR &
 
 script:
-    - xvfb-maybe npm run build
+    - xvfb-maybe npm install
```

Comment:
Okay, I detached tests from the build process. Should now all be really explicit.